contracts/components/staking/FortaStaking.sol

```diff
@@ -104,9 +102,10 @@ contract FortaStaking is BaseComponentUpgradeable, ERC1155SupplyUpgradeable, Sub
     event Released(uint8 indexed subjectType, uint256 indexed subject, address indexed to, uint256 value);
     event DelaySet(uint256 newWithdrawalDelay);
     event TreasurySet(address newTreasury);
-    event StakeParamsManagerSet(address indexed newManager);
+    event StakeHelpersConfigured(address indexed subjectManager, address indexed allocator);
```

Comment:
Please note it says "manager" here and I think it will have to be renamed to "gateway".

Reply:
Noted

Reply:
And fixed